integration-tests/src/test/resources/sitconfig/java/SitConfigTests.java

```diff
@@ -0,0 +1,554 @@
+// Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.*;
+import java.util.HashMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.management.MBeanServerConnection;
+import javax.management.MalformedObjectNameException;
+import javax.management.ObjectName;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.sql.DataSource;
+import weblogic.diagnostics.descriptor.WLDFHarvestedTypeBean;
+import weblogic.diagnostics.descriptor.WLDFInstrumentationMonitorBean;
+import weblogic.diagnostics.descriptor.WLDFResourceBean;
+import weblogic.j2ee.descriptor.wl.JDBCConnectionPoolParamsBean;
+import weblogic.j2ee.descriptor.wl.JDBCDataSourceBean;
+import weblogic.j2ee.descriptor.wl.JDBCDriverParamsBean;
+import weblogic.j2ee.descriptor.wl.JMSBean;
+import weblogic.j2ee.descriptor.wl.UniformDistributedTopicBean;
+import weblogic.management.ManagementException;
+import weblogic.management.configuration.*;
+import weblogic.management.jmx.MBeanServerInvocationHandler;
+import weblogic.management.mbeanservers.domainruntime.DomainRuntimeServiceMBean;
+import weblogic.management.mbeanservers.edit.ConfigurationManagerMBean;
+import weblogic.management.mbeanservers.edit.EditServiceMBean;
+import weblogic.management.mbeanservers.runtime.RuntimeServiceMBean;
+import weblogic.management.runtime.DomainRuntimeMBean;
+import weblogic.management.runtime.ServerRuntimeMBean;
+
+public class SitConfigTests {
+
+  private MBeanServerConnection domainRuntimeMbs, editMbs, runtimeMbs;
+  private JMXConnector jmxConnector;
+  private static ObjectName service;
+  private DomainRuntimeMBean domainRuntimeMBean;
+  private DomainRuntimeServiceMBean domainServiceMBean;
+  private RuntimeServiceMBean runtimeServiceMBean;
+  private ServerRuntimeMBean serverRuntime;
+  private EditServiceMBean editServiceMBean;
+
+  private DomainMBean editdomain, editTree;
+  private ConfigurationManagerMBean cfgMgr;
+  private static final String JNDI = "/jndi/";
+
+  private final String adminHost;
+  private final String adminPort;
+  private final String adminUser;
+  private final String adminPassword;
+
+  public static void main(String args[]) throws Exception {
```

Comment:
Is this really run as a main class?  I thought all of the tests ran as JUnit tests.

Reply:
Okay, that needs to be documented here

Reply:
add the documentation

Reply:
Added documentation